desktop/wsl/index.md

```diff
@@ -0,0 +1,86 @@
+---
+description: Turn on the Docker WSL 2 backend and get to work using best practices, GPU support, and more in this thorough guide. 
+keywords: wsl, wsl2, installing wsl2, wsl installation, docker wsl2, wsl docker, wsl2 tech preview, wsl install docker, install docker wsl, how to install docker in wsl
+redirect_from:
+- /docker-for-windows/wsl/
+- /docker-for-windows/wsl-tech-preview/
+- /desktop/windows/wsl/
+title: Docker Desktop WSL 2 backend on Windows
+---
+
+Windows Subsystem for Linux (WSL) 2 is a full Linux kernel built by Microsoft, which lets Linux distributions run without managing virtual machines. With Docker Desktop running on WSL 2, users can leverage Linux workspaces and avoid maintaining both Linux and Windows build scripts. In addition, WSL 2 provides improvements to file system sharing and boot time.
+
+Docker Desktop uses the dynamic memory allocation feature in WSL 2 to improve the resource consumption. This means Docker Desktop only uses the required amount of CPU and memory resources it needs, while allowing CPU and memory-intensive tasks such as building a container, to run much faster.
+
+Additionally, with WSL 2, the time required to start a Docker daemon after a cold start is significantly faster. 
+
+## Prerequisites
+
+Before you turn on the Docker Desktop WSL 2 feature, ensure you have:
+
+- At a minimum WSL version 1.1.3.0., but ideally the latest version of WSL to [avoid some Docker Desktop functionalities not working](best-practices.md).
+- Windows 10, version 21H2 or later, or Windows 11, version 21H2 or later. For more information, see [System requirements](https://docs.docker.com/desktop/install/windows-install/#system-requirements).
+- Installed the WSL 2 feature on Windows. For detailed instructions, refer to the [Microsoft documentation](https://docs.microsoft.com/en-us/windows/wsl/install-win10){:target="_blank" rel="noopener" class="_"}.
+
```

Comment:
There used to be a step here about downloading the Linux kernel update package, but I have removed this as this is now managed automatically by Windows updates https://learn.microsoft.com/en-us/windows/wsl/kernel-release-notes